ui-core/src/surveyjs/surveyJsStyle.scss

```diff
@@ -25,7 +24,6 @@
 }
 
 .sd-html p:only-child {
-  padding-top: 2em;
```

Comment:
We no longer need to add this padding - surveyjs html blocks seem to have been updated to include this padding by default